package.json

```diff
@@ -23,12 +23,14 @@
     "remove-tsd-loader-typings": "rimraf node_modules/typescript-simple-loader/typescript-simple-loader.d.ts",
     "tsd-update": "npm run remove-tsd-loader-typings && tsd reinstall --overwrite",
     "postinstall": "npm run tsd-update && tsd install && tsd link",
-    "start": "rimraf build && webpack --watch"
+    "start": "rimraf build && webpack --watch",
+    "test": "webpack --config webpack.test.config.js && browserify build/test.js | node_modules/tape-run/bin/run.js | node_modules/tap-spec/bin/cmd.js"
```

Comment:
:checkered_flag: scratch that, use this instead: `"test": "webpack --config webpack.test.config.js && cat build/test.js | tape-run | tap-spec"`